src/index.html

```diff
@@ -7,12 +7,54 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page">
+    <section
+      data-qa="card"
+      class="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="Macbook"
+      />
+
+      <h2 class="card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">Product code: 195434</p>
```

Comment:
The p tag is used for paragraphs. Here you are displaying a product code, which is not really a paragraph. You could use a div or a span instead.